tests/multi_bluetooth/ble_gatt_data_transfer.py

```diff
@@ -29,35 +29,53 @@
 SERVICE = (SERVICE_UUID, (CHAR_CTRL, CHAR_RX, CHAR_TX))
 SERVICES = (SERVICE,)
 
-waiting_event = None
-waiting_data = None
-ctrl_value_handle = 0
-rx_value_handle = 0
-tx_value_handle = 0
+waiting_events = {}
 
 
 def irq(event, data):
-    global waiting_event, waiting_data, ctrl_value_handle, rx_value_handle, tx_value_handle
     if event == _IRQ_CENTRAL_CONNECT:
         print("_IRQ_CENTRAL_CONNECT")
+        waiting_events[event] = data[0]
     elif event == _IRQ_CENTRAL_DISCONNECT:
         print("_IRQ_CENTRAL_DISCONNECT")
     elif event == _IRQ_GATTS_WRITE:
         print("_IRQ_GATTS_WRITE")
+        waiting_events[
+            (
+                event,
+                data[1],
```

Comment:
if you remove the comma on this line then I think black will put everything here on one line (there are also other cases of this below)